### ISSUE TITLE
add time stamp for recovery services log messages

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-ios#194]
subj

**Comment by vdelendik** [comment]
fixed as part of #208 fix: https://github.com/zebrunner/mcloud-ios/pull/210/files#diff-a471c445a7204680782fc16eb72dbd712fe57175d0db0eb0fb0dbfa6ada5b582R696

starting from now in agents.log we should see datetime:
```
[03/03/2023 15:39:36] Stop services for ECOMM_iPhone_12_mini (00008101-000470C91100001E)
[03/03/2023 15:40:16] Recover services for ECOMM_iPhone_XR (00008020-000D39913E13802E)
```

**Comment by dhreben** [comment]
Fixed
